samples/ResourceOwnerPasswordFlow/DeveloperNotes/ConformanceProfiles/fiddler-request-OP-Response-Missing.txt

```diff
@@ -0,0 +1,7 @@
+﻿POST http://localhost:50000/connect/token HTTP/1.1
```

Comment:
Out of curiosity, have you considered adding a little console project using `HttpClient` to retrieve an access token?